templates/hmc/values.yaml

```diff
@@ -7,7 +7,8 @@ admissionWebhook:
   certDir: "/tmp/k8s-webhook-server/serving-certs/"
 
 controller:
-  defaultOCIRegistry: "oci://ghcr.io/mirantis/hmc/charts"
+  defaultRegistryURL: "oci://ghcr.io/mirantis/hmc/charts"
+  defaultRepoType: "oci"
```

Comment:
It would be nice to automatically discover the type based on URL scheme. what do you think?

Reply:
I need to make sure the OCI spec does not allow for URLs to start with `http/https` too, but I'd be game, but if that's the case -- it makes me wonder why Flux didn't adopt URL scheme and introduced type in the first place.

Reply:
There's nothing in the OCI spec that says `oci://` should be the URI scheme, it simply notes that OCI operates over `http://`, my only concern with changing this is the what if some customer comes along and uses an OCI type repo and they enter their registry URL as `https://` because that's what makes sense to them.

Reply:
It seems Flux makes an assumption based on URL scheme and provides useful errors if an OCI typed repository is used with an `http/https` URL scheme so I think it's probably fine for us to assume as well.